lib/data/external/AwsClient.js

```diff
@@ -125,7 +125,7 @@ class AwsClient {
             Bucket: this._awsBucketName,
             Key: key,
             VersionId: dataStoreVersionId,
-            Range: range,
+            Range: range ? `bytes ${range[0]}-${range[1]}` : null,
```

Comment:
The proper HTTP format for a full range request is `Range: bytes=A-B`, so I assume the AWS client expects the same.